packages/components/focus-trap/src/focus-trap.vue

```diff
@@ -168,11 +168,9 @@ export default defineComponent({
       if (focusLayer.paused || !trapContainer) return
 
       if (props.trapped) {
-        if (
-          !trapContainer.contains(
-            (e as FocusEvent).relatedTarget as HTMLElement | null
-          )
-        ) {
+        const relatedTarget = (e as FocusEvent)
+          .relatedTarget as HTMLElement | null
+        if (trapContainer != null && !trapContainer.contains(relatedTarget)) {
```

Comment:
```suggestion
import { isNil } from 'lodash-unified'
        if (!isNil(trapContainer) && !trapContainer.contains(relatedTarget)) {
```

Reply:
Since `isNil` cannot tell typescript about whether this `trapContainer` is null or not, we will just skip it.